hack/test-extended/forcepull/run.sh

```diff
@@ -0,0 +1,141 @@
+#!/bin/bash
```

Comment:
just my lack of knowledge about the extended test framework, but..what actually runs this script then a PR is tagged for extended tests?  is the vagrant plugin going to explicitly invoke this?  or is there a wrapper test-extended.sh script that invokes all these?